content/links.md

```diff
@@ -408,3 +471,52 @@ instance_groups:
       - port
       - url
 ```
+___
+
+## Possible Link Scenarios
+
+The rules for uniqueness are described in the tables below:
+
+Providers can be specified in any way you want as long as it is **not** consumed.
+
+| Provider Name | Provider Type | Consumer Type | Allowed |
+|---------------|---------------|---------------|---------|
+| Same          | Same          | No consumer   | True    |
+| Same          | Different     | No consumer   | True    |
+| Different     | Same          | No consumer   | True    |
+| Different     | Different     | No consumer   | True    |
+
+Providers with different names or types can always be **explicitly** consumed without conflicts.
+
+| Provider Name | Provider Type | Consumer Type | Allowed |
+|---------------|---------------|---------------|---------|
+| Same          | Same          | Explicit      | False   |
+| Same          | Different     | Explicit      | True    |
+| Different     | Same          | Explicit      | True    |
+| Different     | Different     | Explicit      | True    |
+
+Providers with different types can always be **implicitly** consumed without conflicts.
+
+| Provider Name | Provider Type | Consumer Type | Allowed |
+|---------------|---------------|---------------|---------|
+| Same          | Same          | Implicit      | False   |
+| Same          | Different     | Implicit      | True    |
+| Different     | Same          | Implicit      | False   |
+| Different     | Different     | Implicit      | True    |
+
+___
+
+## Links FAQ
+
+Q: What characters are valid for link names?<br/>
+A: You should try to use unicode characters for naming. However you can not begin with a colon (`:`) since the ruby YAML parser treats it as a symbol.
```

Comment:
I would lean towards saying we recommend links names to be ascii letters and numbers only, even if we support wider spectrum. What do you think ?